PythonCourse/Dictionaries/nokia_3310_numbers.py

```diff
@@ -0,0 +1,10 @@
+buttons = {'.,?!:': '1', 'ABC': '2', 'DEF': '3', 'GHI': '4', 'JKL': '5', 
+    'MNO': '6', 'PQRS': '7', 'TUV': '8', 'WXYZ': '9', ' ': '0'
+    }
+
+text = input().upper()
+for el in text:
+    for key, value in buttons.items():
```

Comment:
redo using dictionaries

Reply:
https://stackoverflow.com/questions/2974022/is-it-possible-to-assign-the-same-value-to-multiple-keys-in-a-dict-object-at-onc